gatsby/src/reports/reports.js

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from "react"
+import { StaticQuery, graphql } from "gatsby"
+import Layout from "../layout/layout"
+import Accordion from "../components/accordion"
+
+class ReviewReport extends React.Component {
+  render() {
+    return (
+      <StaticQuery
+        query={graphql`
+          query {
+            allReviewedDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { ne: null } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+            staleDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { lt: "2020-01-01" } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+            unreviewedDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { eq: null }, title: { ne: "" } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+          }
+        `}
+        render={data => {
+          const [search, setSearch] = useState("")
+          const reviewedPages = data.allReviewedDocs.edges
+          const reviewedTertiaryPages = data.allReviewedDocs.edges.filter(
+            page => {
+              return page.node.fields.slug.match(/\/guides(\/[a-z,\-]*){2}/)
+            }
+          )
+          //console.log("Tertiary Pages: ", tertiaryPages) // For debugging
+          const oldPages = data.staleDocs.edges
+          const unreviewedPages = data.unreviewedDocs.edges
+          //console.log("Unreviewed Docs Array: ", unreviewedPages) // For debugging
+          const unreviewedTertiaryPages = data.unreviewedDocs.edges.filter(
+            page => {
+              return page.node.fields.slug.match(/\/guides(\/[a-z,\-]*){2}/)
+            }
+          )
```

Comment:
These constants refer to sub-arrays from the queries above, reducing the amount of drilling down we have to repeat. Additionally, the `Tertiary` constants filter for just those pages with slugs that match the "/guides/*/*` pattern. This is used to differentiate between all docs and guides, and the pages of a multipage guide which we may count as a single entry for some reporting.